### PR TITLE
created Huralain.md

### PR DESCRIPTION
**Pull request by huralain**
https://raw.githack.com/huralain/treehouses.github.io/New_Branch_Huralain/#!pages/vi/profiles/Huralain.md

### CODE REVIEW

**Comment by dogi** [comment]
> [raw.githack.com/huralain/treehouses.github.io/New_Branch_Huralain/pages/vi/profiles/Huralain.md](https://raw.githack.com/huralain/treehouses.github.io/New_Branch_Huralain/pages/vi/profiles/Huralain.md)

@huralain above link is not working the right way ...

but otherwise very good start ;)

**Comment by CalebProvost** [comment]
Your link needs the hashbang (`#!`) before the `pages`: https://raw.githack.com/huralain/treehouses.github.io/New_Branch_Huralain/#!pages/vi/profiles/Huralain.md

File: pages/vi/profiles/Huralain.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+My name is **Huralain Mohamed** and I am located in **El Paso, TX, USA**. My current timezone is **MT(-6): Mountain Time**. I am currently operating on a MBP - **macOS Mojave version 10.14.6**.
+
+Just so you can get to know a little bit about me, I am 26 years old and a US Army veteran. I have been working in the tech industry for 9 years, however, I started learning how to code this year. I am excited to start working with you all at OLE to further expand my skillset.
+
+Some fun facts about me are:
++ Pizza is my favorite food.
++ I've been to 15+ countries.
++ I was born in LA, but raised in Boston.
++ I'm a fashion designer as well as a coder.
```
Comment (CalebProvost) [inline pages/vi/profiles/Huralain.md:11]: Your "Fun facts" section's list isn't rendering correctly. 

Other than that. Good job!

**Review by CalebProvost — CHANGES_REQUESTED** [review]

File: pages/vi/profiles/Huralain.md (hunk 1 above)
Comment (huralain) [reply pages/vi/profiles/Huralain.md:11]: Thank you so much Caleb! I'll fix that issue :)

**Comment by huralain** [comment]
> > [raw.githack.com/huralain/treehouses.github.io/New_Branch_Huralain/pages/vi/profiles/Huralain.md](https://raw.githack.com/huralain/treehouses.github.io/New_Branch_Huralain/pages/vi/profiles/Huralain.md)
> 
> @huralain above link is not working the right way ...
> 
> but otherwise very good start ;)

Hello Dogi, thank you so much! I corrected the link to get it to show properly.

**Review by huralain — COMMENTED** [review]
thank you I made the changes!

**Review by CalebProvost — APPROVED** [review]

**Comment by huralain** [comment]
> Your link needs the hashbang (`#!`) before the `pages`: https://raw.githack.com/huralain/treehouses.github.io/New_Branch_Huralain/#!pages/vi/profiles/Huralain.md

Thank you Caleb! I updated that.

**Review by XavierElon — APPROVED** [review]
Good job!

**Comment by vmnet8** [comment]
Good job!

**Pull request merged**